README.md

```diff
@@ -1,24 +1,119 @@
-# README
+# Private-Events
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Table of Contents
+  <br />
 
-Things you may want to cover:
+* [About the Project](#about-the-project)
+* [Built With](#built-with)
+* [Contributing](#contributing)
+* [Contact](#authors)
+* [Acknowledgements](#acknowledgements) 
 
-* Ruby version
+#
+<br />
 
-* System dependencies
+[![forthebadge](https://forthebadge.com/images/badges/made-with-ruby.svg)](https://forthebadge.com) </br>
+![rails](https://img.shields.io/badge/Ruby_on_Rails-CC0000?style=for-the-badge&logo=ruby-on-rails&logoColor=white)
 
-* Configuration
+<br />
 
-* Database creation
+#
 
-* Database initialization
 
-* How to run the test suite
+<!-- ABOUT THE PROJECT   -->
+## About The Project
 
-* Services (job queues, cache servers, search engines, etc.)
+The Private Event project sets up an initial rails app containing a User, Event databases. This page can log in users and create events for each of them. As well can let users attend others events.
+
+## Screenshot Test
+#
+<p align="center">
+  <img height="auto" src="Screenshot.png">
+</p>
+
+#
+
+### Built With
+
+* [RUBY]
+* [RAILS]
+* [DEVISE]
+* [RSPEC&CAPIBARA]
+
+## Contributing
+
+Contributions make the open-source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+## Prequisites
+
+Ruby & Rails installed on your local machine.
+bundle installed on your machine
+Terminal or similar to execute the program.
+
+
+## Getting Started
+
+
+## Clone project
+
+- To get a local copy up and running follow these simple example steps.
+- Clone this repository with git clone ```https://github.com/sergiocortessat/Rails_Private_Events/tree/feature``` using your terminal or command line.
+- Change to the project directory by entering: ```cd Private_Events``` in the terminal.
+
+
+## How to use the Private-Event app
+1. run the command ```bundle install```   This will install the required dependencies
+2. run the command ```rails db:create```. This will initialize the necessary migration files
+3. run the command ```rails db:migrate``` 
```

Comment:
- Add instruction for installing webpack using `yarn` to get rid of the error: ` Webpacker::Manifest::MissingEntryError in Events#index`